power-pay-backend/src/test/java/com/adorsys/gis/powerpay/powerpaybackend/PersonMapper.java

```diff
@@ -0,0 +1,14 @@
+package com.adorsys.gis.powerpay.powerpaybackend;
+
+// PersonMapper.java
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.factory.Mappers;
+
+@Mapper
```

Comment:
Add the type component to this mapper, as we're working with Spring Boot.

https://www.baeldung.com/mapstruct#Injection